src/sentry/relay/config/__init__.py

```diff
@@ -51,6 +51,7 @@
 # NOTE: These features must be sorted or the tests will fail!
 EXPOSABLE_FEATURES = [
     "organizations:continuous-profiling",
+    "organizations:continuous-profiling-allowed-ingest",
```

Comment:
What is the plan here after exposing this flag to relay?

Reply:
Sorry, I meant how do you plan on doing that? If you merge this + the relay PR, by default all continuous profiles will be rejected.

Reply:
The issue with this is that we're now relying on the timing of a deploy to enable these flags. Ideally, all the code to handle things should already be in production, and on the day of, we should just need to flip a few options to handle things

Reply:
well, if we don't want to rely on the timing of the deploy I can slightly refactor the FeatureHandler to make it an options backed Feature.

I'll add a couple of options:
* stop_non_beta_ingestion (bool)
* beta_orgs (list of orgs)

Now, on the 14th. we open an Option Automator PR with `stop_non_beta_ingestion` set to True and `beta_orgs` set to the desired list of orgs.

This way I could start merging the Relay and Getsentry changes and we'll only flip a few options when the time will come.

Reply:
Good point, maybe it's easier afterall to just use the existing feature flag and just tune the handler?

Reply:
> Good point, maybe it's easier afterall to just use the existing feature flag and just tune the handler?

Yes please.